.github/workflows/ci.yml

```diff
@@ -46,3 +47,4 @@ jobs:
             arduino-cli compile --fqbn OpenCR:OpenCR:OpenCR /github/home/Arduino/libraries/micro_ros_arduino-0.0.1/examples/micro-ros_publisher -v
             arduino-cli compile --fqbn teensy:avr:teensy31 /github/home/Arduino/libraries/micro_ros_arduino-0.0.1/examples/micro-ros_publisher -v
             arduino-cli compile --fqbn teensy:avr:teensy41 /github/home/Arduino/libraries/micro_ros_arduino-0.0.1/examples/micro-ros_publisher -v
+            arduino-cli compile --fqbn arduino:samd:arduino_zero_native /github/home/Arduino/libraries/micro_ros_arduino-0.0.1/examples/micro-ros_publisher -v
```

Comment:
CI tests are missing here as the upstream version of Arduino Due core doesn't support static libraries. Here is a relevant PR:
https://github.com/arduino/ArduinoCore-sam/pull/115

Reply:
Ok, I'm curious to know why they allow precompiled linking and dont have to support in their own boards